dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -26,7 +26,7 @@ debs/{{ deb }}{{' '}}
 RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
 RUN rm -rf /debs
 
-COPY ["start.sh", "orchagent.sh", "/usr/bin/"]
+COPY ["start.sh", "orchagent.sh", "swssconfig.sh", "/usr/bin/"]
```

Comment:
@stcheng: I've been using this naming convention for supervisord. Basically, if a process requires more than one command to start properly, I've been wrapping them in a shell script named after the process. I think "config.sh" is not descriptive enough and can't be used as a convention. Maybe "swssconfig_start.sh"?